src/applications/mhv/secure-messaging/components/MessageActionButtons/MoveMessageToFolderBtn.jsx

```diff
@@ -110,10 +110,8 @@ const MoveMessageToFolderBtn = props => {
         data-testid="message-actions-buttons-modal"
       >
         <VaModal
-          uswds={false}
           id="move-to-modal"
           data-testid="move-to-modal"
-          large
           modalTitle="Move to:"
           onCloseEvent={closeModal}
           visible={isMoveModalVisible}

```

Comment:
I saw that the buttons on this modal were slightly off in 320x dimensions. It might be beneficial to check the other modals too.